api/app/tests/apps/sigmax/stuf_protocol/outgoing/test_pdf.py

```diff
@@ -24,6 +24,8 @@ def test_render_html(self):
         signal = factories.SignalFactoryWithImage.create(
             incident_date_start=timezone.now(),
             extra_properties=extra_properties_data,
+            category_assignment__category__parent__name='Wegen, verkeer, straatmeubilair',
+            category_assignment__category__name='lantaarnpaal straatverlichting',
```

Comment:
Reading through this unittest I think a test is missing; the unittest does not check for the presence or absense of `extra_properties` data (so the filtering that this PR implements is not tested).

We should expand the test a little:
* by checking that `Niet gevaarlijk` is present in the data
* and by adding data for another category in the `extra_properties_data` and making sure it is not rendered to the HTML

Reply:
Done!